pkg/actors/engine/engine.go

```diff
@@ -74,6 +76,12 @@ type engine struct {
 
 	lock  *fifo.Mutex
 	clock clock.Clock
+
+	actorCallOpts actorCallOptions
+}
+
+type actorCallOptions struct {
+	maxRequestBodySize int
```

Comment:
I don't it's necessary for us to have a new struct here for this

Reply:
Changed